### PR TITLE
Implemented Schedule Change Accept / AckScheduleChangeReq/Rsp

### PR DESCRIPTION
**Pull request by creduo**
Due to recent global event, there are too many cancellations and schedule changes, this seems needed in some cases.

calling AckScheduleChangeReq will update seat statuses from TK to HK.

### CODE REVIEW

File: docs/Air.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ The Air workflow allows you to do what most travel agents did in the past and wh
 * [.getTickets(params)](#getTickets)
 * [.cancelTicket(params)](#cancelTicket)
 * [.cancelBooking(params)](#cancelBooking)
+* [.ackScheduleChange(params)](#ackScheduleChange)
```
Comment (dchertousov) [inline docs/Air.md:29]: Please rename to `acknowledge` or `accept` to make it more readable.
I needed to consult vocabulary for abbreviation meaning :)

File: src/Services/Air/AirParser.js [hunk 2]
```diff
@@ -1327,6 +1332,7 @@ module.exports = {
   AIR_GET_TICKETS_ERROR_HANDLER: airGetTicketsErrorHandler,
   AIR_CANCEL_TICKET: airCancelTicket,
   AIR_CANCEL_PNR: airCancelPnr,
+  ACK_SCHEDULE_CHANGE: ackScheduleChange,
```
Comment (dchertousov) [inline src/Services/Air/AirParser.js:1335]: should be still prepended by `AIR`

File: docs/Air.md (hunk 1 above)
Comment (creduo) [reply docs/Air.md]: Accept sounds more natural, but the uAPI named ack, so renamed to acknowledgeScheduleChange

File: src/Services/Air/AirParser.js (hunk 2 above)
Comment (creduo) [reply src/Services/Air/AirParser.js]: Added AIR_ prefix

File: docs/Air.md (hunk 1 above)
Comment (dchertousov) [reply docs/Air.md]: We don't always follow api naming routine, you know :)
In your another PR you took traveler as it's in uAPI, but we had passengers already, so you may keep accept. I find it more suitable too.

Comment (creduo) [reply docs/Air.md]: So, should we name it to acceptScheduleChange?

Comment (dchertousov) [reply docs/Air.md]: I would prefer it.

Comment (creduo) [reply docs/Air.md]: renamed it to acceptScheduleChange :)

**Pull request closed without merging**